src/server/node_manager.c

```diff
@@ -7114,11 +7114,27 @@ set_nodes(void *pobj, int objtype, char *execvnod_in, char **execvnod_out, char
 				return (PBSE_UNKNODE);
 			}
 
-                        if (svr_init == FALSE && (pnode->nd_state & (INUSE_DOWN | INUSE_STALE))) {
-                                   free(phowl);
-                                   free(execvncopy);
-                                   return (PBSE_BAD_NODE_STATE);
-                        }
+                	if ((pnode->nd_state & (INUSE_DOWN | INUSE_STALE))&& (svr_init == FALSE)) {
+                            if (objtype == JOB_OBJECT) {
+				free(phowl);
+                                free(execvncopy);
+                                return (PBSE_BAD_NODE_STATE);
```

Comment:
I think these two lines are indented further than they should be.  Either that or the first free() is not indented enough.